packages/harmony/src/foundations/reset/reset.css

```diff
@@ -127,3 +127,11 @@ table {
   border-collapse: collapse;
   border-spacing: 0;
 }
+
+/* Ensure fonts are consistent across browser environments */
+:root {
```

Comment:
Woah this is super cool. Thank you a ton for digging into this, will share with the team tomorrow as a heads up